static/js/thing-detail-layout.js

```diff
@@ -0,0 +1,36 @@
+(function() {
+
+function ThingDetailLayout(elements) {
```

Comment:
nit: Could this just be ThingDetail rather than ThingDetailLayout?

Reply:
As a function this lays out a collection of a thing's detail circles. Admittedly the only reason this exists is that it was necessary to use d3, although I like the idea of being able to modularly change the layout from wide to tall (or whatever ends up being necessary)